.github/workflows/build-ws-container.yml

```diff
@@ -27,8 +38,43 @@ jobs:
           git checkout $(git rev-list --tags --max-count=1)
           git describe
 
-      - name: Log into container registry
-        run: $RUNC login -u ${{ secrets.QUAY_BOTUSER }} -p ${{ secrets.QUAY_TOKEN }} quay.io
+      - name: Build ws container
+        id: build
+        run: containers/ws/release.sh cockpit-ws-${{ matrix.build.label }}.tar
+        env:
+          IMAGE_TAG: localhost/cockpit-ws:release-${{ matrix.build.label }}
+
+      - name: Save image to artifacts
+        uses: actions/upload-artifact@v4
+        with:
+          name: cockpit-ws-${{ matrix.build.label }}
+          path: cockpit-ws-${{ matrix.build.label }}.tar
```

Comment:
This is really nice! Much better than having to upload the temporary container image to the registry.